aliyun-net-sdk-core/Auth/InstanceProfileCredentialsProvider.cs

```diff
@@ -67,6 +67,7 @@ AlibabaCloudCredentials AlibabaCloudCredentialsProvider.GetCredentials()
                 {
                     // Use the current expiring session token and wait for next round
                     credentials.SetLastFailedRefreshTime();
+                    e = null;
```

Comment:
应该是 catch (ClientException e) => catch (ClientException) 吧。

Reply:
```suggestion
                catch (ClientException e)
                {
                    // Use the current expiring session token and wait for next round
                    credentials.SetLastFailedRefreshTime();
                    e = null;
                }
```

原代码已经catch过了，这个catch是抛出异常，而是刷新。
warning提示是这个e变量没有使用